data/mods/Generic_Guns/gg_vehicle_override.json

```diff
@@ -4,6 +4,7 @@
     "type": "vehicle_part",
     "name": "mounted medium machine gun",
     "item": "m240",
+    "description": "IF YOU ARE SEEING THIS IT IS A BUG.",
```

Comment:
You've got me there, those are not needed, and were never intended, and have been removed.